### DISCUSSION TITLE
Mend app VS self hosting (pricing, data retention etc)

### DISCUSSION THREAD

**Discussion by henrikix** [renovatebot/renovate#29598] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

None

### Please tell us more about your question or problem

Hi!

Could you perhaps help me out/elaborate things for me a bit.

I am interested in automating dependency updates for my repositories on **Bitbucket Cloud** and came upon Mend/Renovate.

Now my confusion lies in the following:

Your readme says `For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.` but here you mention the Mend app. So which one should I use?

When I navigated to your [pricing page](https://www.mend.io/pricing/) then I see large amounts being charged for it but again, here you mention a free plan.

What is the difference between self-hosting, the free app and the paid app?
- How is data handled for each one?
- What are the limits for each one?
- How do you benefit in providing a free plan?
- What to keep in mind/best practices for self-hosting?
- Any other topics you might elaborate on

Just to specify - I am not interested on using it on my personal repositories but work ones so how the data is handle is quite important.

Thank you!

**Comment by rarkins** [comment, marked as answer]
> Your readme says `For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.` but here you mention the Mend app. So which one should I use?

For Bitbucket Cloud, we recommend you install the hosted Mend app and select the free Renovate plan.

> When I navigated to your [pricing page](https://www.mend.io/pricing/) then I see large amounts being charged for it but again, here you mention a free plan.

Renovate is free for the hosted apps (github.com, Bitbucket cloud)

> What is the difference between self-hosting, the free app and the paid app?

In self-hosting you host and you need to pay for compute and monitor it. In the app it's hosted by Mend. The free plan includes Renovate only. The paid plan in the app includes AppSec capabilities such as OSS security and licensing compliance, first party code scanning, etc.

> * How is data handled for each one?

I'm going to assume for now that you'll use the free plan. We have security description here which includes some specifics on the app: https://docs.renovatebot.com/security-and-permissions/

If you need more details than that, then you sound like the person who needs a paid app. We don't heavily push it, but you'd be most welcomed by the Mend.io sales team if you'd like to make an inquiry. We can't spend time answering questions like these from every free user or we couldn't afford to run the service.

> * What are the limits for each one?

Right now there isn't much in the way of limits other than that it's Renovate only. Towards end of year we'll likely have some tiered plans for those who need e.g. concurrent jobs on multiple repositories, or faster machines than we have currently.

> * How do you benefit in providing a free plan?

1. Referrals/brand recognition for Mend.io
2. Data gathered on it about open source packages is used to build our Merge Confidence Database

> * What to keep in mind/best practices for self-hosting?

Read the docs, and you're on your own.

**Reply by henrikix** [reply]
I appreciate the quick and informative reply, thanks!

**Reply by HonkingGoose** [reply]
> - What to keep in mind/best practices for self-hosting?

I recommend that you read the [Renovate docs, Upgrade best practices](https://docs.renovatebot.com/upgrade-best-practices/) to learn how best to manage updates. Updating regularly is extra important when you're self-hosting.
